public/js/ui.js

```diff
@@ -120,6 +120,30 @@ if (!linz) {
 
         });
 
+        // add ability to open URL in a modal
+        $('[data-linz-modal]').click(function () {
+
+            if ($(this).attr('data-linz-disabled') === 'true') {
+                return;
+            }
+
+            var queryObj = $(this),
+                url = queryObj[0].nodeName === 'BUTTON' ? queryObj.attr('data-href') : queryObj.attr('href');
+
+            $('#linzModal').modal().load(url, function () {});
```

Comment:
@smebberson, we have agreed that including ```data``` and ```complete``` don't provide any benefit for this feature. However, I've added the ability to define a ```callback``` function that will execute after the modal is shown. Below are 2 examples of the modal DSL:

```
# modal with a callback
modal : {
    callback: 'jsFunctionName'
}

# modal with no callback
modal : true
```